src/App.vue

```diff
@@ -26,19 +43,27 @@ export default {
           // Request account access
           await window.ethereum.request({ method: "eth_requestAccounts" });
 
-          let web3 = new Web3(window.ethereum);
-          let accounts = await web3.eth.getAccounts();
-          let balance = await web3.eth.getBalance(accounts[0]);
-          this.balance = balance;
-          this.login = true;
+          //update login state and logins status in local storage
+          this.isLoggedIn = true;
+          if (window.localStorage) {
+            window.localStorage.setItem("isLoggedIn", true);
+          }
         } catch (err) {
           // User denied access
-          console.info(" User denied access");
+          console.error("Error while connecting to Metamask");
         }
       } else {
         alert("Unable to detect MetaMask");
       }
     },
+    async getBalance() {
+      let web3 = new Web3(window.ethereum);
```

Comment:
best to use`const` at first priority
If not possible with `const` go for `let`
Last option `var`